WcaOnRails/db/migrate/20231010152057_create_attendee_payment_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAttendeePaymentRequests < ActiveRecord::Migration[7.0]
+  def change
+    create_table :attendee_payment_requests do |t|
+      t.string :attendee_id
+      t.string :payment_id
```

Comment:
What is this ID referencing? Is it a Stripe API ID? If yes, we should rather point to our own Stripe log tables which can then be used to retrieve an ID

Reply:
it's an intent.id, so I think yes?